docs/new_instrument.rst

```diff
@@ -663,20 +702,21 @@ be set up as follows:
         return
 
 
+.. _rst_test-temp:
+
 Supported Instrument Templates
 ------------------------------
 
-Instrument templates may be found at ``pysat.instruments.templates``
-and supporting methods may be found at ``pysat.instruments.methods``.
+Instrument templates may be found at :py:mod:`pysat.instruments.templates`
+and supporting methods may be found at :py:mod:`pysat.instruments.methods`.
 
 General
 ^^^^^^^
 
 A general instrument template is included with pysat,
-``pysat.instruments.templates.template_instrument``,
-that has the full set
-of required and optional methods, and docstrings, that may be used as a
-starting point for adding a new instrument to pysat.
+:py:mod:`pysat.instruments.templates.template_instrument`, that has the full set
```

Comment:
This one works. The .templates and .methods above don't.

Reply:
I am guessing it's because this one has an API entry and the others are displayed differently?

Reply:
👍 I don't see a visual difference on my end. Not a complaint. The domains label stuff is new to me so making you aware in general in case there is something to do.